packages/docs/getting-started/reconnecting-to-network.md

```diff
@@ -0,0 +1,119 @@
+
+```bash
+# On your local machine
+cd celo-accounts-node
+rm -rf geth* && rm static-nodes.json
+```
+
+```bash
+# On your proxy machine
+# Here, you will want to preserve the nodekey so that your enode address doesn't change.
+cd celo-proxy-node
+mv geth/nodekey nodekey
+rm -rf geth* && rm static-nodes.json
+mkdir geth
+mv nodekey geth/nodekey
+```
+
+```bash
+# On your validator machine
+cd celo-validator-node
+rm -rf geth* && rm static-nodes.json
+```
+
+## Restarting the components
+
+### Pulling the latest Celo Docker image
+
+First, pull the Celo image as described at the (Pull the Celo Docker image)[running-a-validator.md#pull-image] section. Be sure to pull the image on your local, validator, proxy, and attestations machines.
+
+### Restart your Accounts node
+
+First, make sure you have the proper environment variables set:
+
+```bash
+# On your local machine
+export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:baklava
+export NETWORK_ID=12219
+```
+
+Follow [these instructions](running-a-validator.md#start-your-accounts-node) to restart your accounts node on your local machine.
+
+### Restart your Proxy node
+
+Follow [these instructions](running-a-validator.md#deploy-a-proxy) to restart your proxy node on your proxy machine.
+
+### Restart your Validator node
+
+In order to restart your Validator, you'll need to have the proxy info environment variables set. If not, follow [these instructions](running-a-validator.md#get-your-proxy's-connection-info) to recover them.
+
+Remember, `PROXY_IP` should be set to the external IP address of your Proxy machine.
+
+```bash
+# On your validator machine
+echo $PROXY_IP
+echo $PROXY_ENODE
+```
+
+Next, follow [these instructions](running-a-validator.md#connect-the-validator-to-the-proxy) to restart your validator node on your validator machine.
+
+### Re-register your Validator
+
+#### Restore any missing environment variables
+
+When you originally set up your validator you should have exported multiple environment variables with information about your Celo accounts.
+
+Please check you have the following environment variables exported in your system:
+
+```bash
+# On your local machine
+echo -e "My Celo Key material:"
+echo CELO_VALIDATOR_ADDRESS=$CELO_VALIDATOR_ADDRESS
+echo CELO_VALIDATOR_GROUP_ADDRESS=$CELO_VALIDATOR_GROUP_ADDRESS
+echo CELO_VALIDATOR_SIGNER_ADDRESS=$CELO_VALIDATOR_SIGNER_ADDRESS
+echo CELO_VALIDATOR_SIGNER_PUBLIC_KEY=$CELO_VALIDATOR_SIGNER_PUBLIC_KEY
+echo CELO_VALIDATOR_SIGNER_SIGNATURE=$CELO_VALIDATOR_SIGNER_SIGNATURE
+echo CELO_VALIDATOR_SIGNER_BLS_PUBLIC_KEY=$CELO_VALIDATOR_SIGNER_BLS_PUBLIC_KEY
+echo CELO_VALIDATOR_SIGNER_BLS_SIGNATURE=$CELO_VALIDATOR_SIGNER_BLS_SIGNATURE
+```
+
+If any of the environment variables are missing, you have two options:
+
+1.  Find the shell when you run the instructions and look for the environment variable missing. You can export it again using the following command:
```

Comment:
Nit:
Replace
>  Find the shell when you run the instructions

With:
Find the shell **where you ran the original instructions**